lng/french.lng.php

```diff
@@ -432,6 +432,7 @@
 $lng['panel']['decending'] = 'descendant';
 $lng['panel']['search'] = 'Rechercher';
 $lng['panel']['used'] = 'utilisé';
+$lng['admin']['dnsrecords'] = 'Own DNS-Records which should be saved in the zonefile<br /><strong>Attention:</strong> The code won\'t be checked for any errors. If it contains errors, webserver might not start again!'; // ToDo: Translante
```

Comment:
Do not add english language strings to another language file. English is the fallback if there is no string in the correspoding language-file.